orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/util/ContextParameterProcessor.groovy

```diff
@@ -60,9 +60,9 @@ class ContextParameterProcessor {
     ContextUtilities.contextFunctionConfiguration.set(contextFunctionConfiguration)
   }
 
-  Map process(Map parameters, Map context, boolean allowUnknownKeys) {
+  Map<String, Object> process(Map<String, Object> parameters, Map<String, Object> context, boolean allowUnknownKeys) {
```

Comment:
added some type information here that made it easier to call the method from non-Groovy context